scripts/Iris.json

```diff
@@ -1,8 +1,7 @@
 {
-    "citation": "R. A. Fisher. 1936. The Use of Multiple Measurements in Taxonomic Problems. and Asuncion, A. & Newman, D.J. (2007). UCI Machine Learning Repository [http://www.ics.uci.edu/~mlearn/MLRepository.html]. Irvine, CA: University of California, School of Information and Computer Science.
-
-",
+    "citation": "R. A. Fisher. 1936. The Use of Multiple Measurements in Taxonomic Problems. and Asuncion, A. & Newman, D.J. (2007). UCI Machine Learning Repository [http://www.ics.uci.edu/~mlearn/MLRepository.html]. Irvine, CA: University of California, School of Information and Computer Science.",
     "description": "Famous dataset from R. A. Fisher",
+    "addendum": "This dataset has been corrected. Information Source: Asuncion, A. & Newman, D.J. (2007). UCI Machine Learning Repository [http://www.ics.uci.edu/~mlearn/MLRepository.html]. Irvine, CA: University of California, School of Information and Computer Science.",
```

Comment:
Since I wasn't here for a large part of "addendum" 's use in the old scripts, I'm not sure if there were any unusual but useful reasons for using it. Off the top of my head, it seems good to have to show extra information or changes made to the data sets, but since it isn't exposed to the user currently, I agree with putting the information for this set in the description for now. I think it is useful for us to have the "addendum" section around though, since we can reference it to see any important changes made to scripts without cluttering up anything displayed to the user that doesn't need to be. It would be an optional parameter for the scripts.